### PR TITLE
docker-forward 0.1 (new formula)

### PR DESCRIPTION
**Pull request by vultron81**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

…ker-machine and creates ssh tunnels for any exposed ports

### CODE REVIEW

**Comment by DomT4** [comment]
I'm a little confused. You said add a new formula for `docker-compose` which already exists, but this formula is for `docker-forward` which is a new submission. Did you mean to imply the latter `depends_on` the former or?

**Comment by vultron81** [comment]
Sorry about that. Trying to add docker-forward not compose.

File: Formula/docker-forward.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      end
+    end
+
+    ENV.prepend_create_path "PYTHONPATH", libexec
+
+    bin.install "docker-forward"
+  end
+
+  test do
+    system "#{bin}/docker-forward" " -h"
```
Comment (rakkesh) [inline Formula/docker-forward.rb]: Change this to `system "#{bin}/docker-forward", "-h"`.

**Comment by rakkesh** [comment]
Please change your commit message to `docker-forward 0.1 (new formula)`.

**Comment by mikemcquaid** [comment]
This is a self-submission of a project less than 24h old so passing on it for now, sorry. Please consider [creating your own tap](https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md#how-to-create-and-maintain-a-tap) instead. Thanks!

**Pull request closed without merging**